test/Misc/testFloat.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swift -parse-stdlib -emit-assembly %s -o - | %FileCheck %s --check-prefix=CHECK-%target-cpu
```

Comment:
Thanks, this is a good test. One problem though: this won't pass on any platforms *other* than s390x because there will be no CHECK lines, and FileCheck will complain about that. The best way I can think to handle this is to provide a dummy prefix that will match on all platforms:

`%FileCheck %s -check-prefix=CHECK -check-prefix=CHECK-%target-cpu`

```swift
// CHECK-NOT: (dummy CHECK line to avoid FileCheck complaints)
```

(Also, no need to name the file "testFloat", since it's already a test file. Maybe "FloatTruncation"?)